CHANGELOG.md

```diff
@@ -1,13 +1,22 @@
 # Changelog
 
-## Unreleased
+## v0.90.0
 Deprecated:
-- Authorization using Role-Based Access Control
+- Authorization using Role-Based Access Control (will be removed in future releases)
 - LDAP & OIDC Embedded Support
```

Comment:
this one won't be introduced in this release too.

Reply:
Not sure I understand, what should I change?